src/checkstyle/checks.xml

```diff
@@ -3,106 +3,181 @@
         "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
         "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
 <module name="Checker">
-    <module name="FileTabCharacter"/>
+    <module name="FileTabCharacter" />
     <module name="NewlineAtEndOfFile">
-        <property name="lineSeparator" value="lf"/>
+        <property name="lineSeparator" value="lf" />
     </module>
     <module name="RegexpMultiline">
-        <property name="format" value="\r"/>
-        <property name="message" value="Line contains carriage return"/>
+        <property name="format" value="\r" />
+        <property name="message" value="Line contains carriage return" />
     </module>
     <module name="RegexpMultiline">
-        <property name="format" value=" \n"/>
-        <property name="message" value="Line has trailing whitespace"/>
+        <property name="format" value=" \n" />
+        <property name="message" value="Line has trailing whitespace" />
     </module>
     <module name="RegexpMultiline">
-        <property name="format" value="\{\n\n"/>
-        <property name="message" value="Blank line after opening brace"/>
+        <property name="format" value="\n\n\n" />
+        <property name="message" value="Multiple consecutive blank lines" />
     </module>
     <module name="RegexpMultiline">
-        <property name="format" value="\n\n\}"/>
-        <property name="message" value="Blank line before closing brace"/>
+        <property name="format" value="\n\n\Z" />
+        <property name="message" value="Blank line before end of file" />
+    </module>
+
+    <module name="RegexpMultiline">
+        <property name="format" value="\{\n\n" />
+        <property name="message" value="Blank line after opening brace" />
+    </module>
+    <module name="RegexpMultiline">
+        <property name="format" value="\n\n\s*\}" />
+        <property name="message" value="Blank line before closing brace" />
     </module>
     <module name="RegexpMultiline">
-        <property name="format" value="\n\n\n"/>
-        <property name="message" value="Multiple consecutive blank lines"/>
+        <property name="format" value="[^;]\s\)+\s*[\{;,]?\s*\n" />
+        <property name="message" value="Whitespace character before closing parenthesis" />
     </module>
     <module name="RegexpMultiline">
-        <property name="format" value="\n\n\Z"/>
-        <property name="message" value="Blank line before end of file"/>
+        <property name="format" value="->\s*\{\s+\}" />
+        <property name="message" value="Whitespace inside empty lambda body" />
+    </module>
+    <module name="RegexpSingleline">
+        <property name="format" value="(class|interface) ([a-zA-Z0-9_])+(&lt;.*&gt;)? (extends|implements)" />
+        <property name="message" value="No new line before extends/implements" />
```

Comment:
Can we remove the checkstyle for "extends a class or interface " where the extend goes on a separate line